modules/apps/27-interchain-accounts/types/account_test.go

```diff
@@ -88,3 +95,83 @@ func (suite *TypesTestSuite) TestGeneratePortID() {
 		})
 	}
 }
+
+func (suite *TypesTestSuite) TestInterchainAccount() {
+	pubkey := secp256k1.GenPrivKey().PubKey()
+	addr := sdk.AccAddress(pubkey.Address())
+	baseAcc := authtypes.NewBaseAccountWithAddress(addr)
+	interchainAcc := types.NewInterchainAccount(baseAcc, "account-owner-id")
```

Comment:
should we use TestOwner for `account-owner-id`?

Reply:
I added a reusable test address to the file. Is that what you meant?